relative-path/src/tests.rs

```diff
@@ -3,8 +3,10 @@
 use super::*;
 
 use std::path::Path;
+use std::prelude::v1::*;
 use std::rc::Rc;
 use std::sync::Arc;
+use std::{format, vec};
```

Comment:
Avoid grouped imports of different modules.

Reply:
These are actually the two macros.
I don't think I can bring in the vec macro without bringing in the module as well.